modin/pandas/series_utils.py

```diff
@@ -616,10 +625,9 @@ def total_seconds(self, *args, **kwargs):
             query_compiler=self._query_compiler.dt_total_seconds(*args, **kwargs)
         )
 
-    def to_pytimedelta(self):
-        return self._query_compiler.default_to_pandas(
-            lambda df: pandas.Series.dt.to_pytimedelta(df.squeeze(axis=1).dt)
-        )
+    def to_pytimedelta(self) -> npt.NDArray[np.object_]:
+        res = self._query_compiler.dt_to_pytimedelta()
+        return res.to_numpy()[:, 0]
```

Comment:
See: https://github.com/modin-project/modin/pull/1542#issuecomment-639483549

Reply:
im not clear on the question

Reply:
@jbrockmendel sorry for not providing more context. I think @prutskov tried implementing this before but he said that there was some incorrect conversion. I tagged that comment to provide more context as to why we defaulted to pandas here.